### PR TITLE
rectified ann.txt (mlpack/doc)

### PR DESCRIPTION
**Pull request by MuLx10**
Rectified misplacement of curly braces in ann.txt

### CODE REVIEW

**Comment by rcurtin** [comment]
@MuLx10 can you justify this change please?  I don't see what was wrong with the previous documentation.

**Comment by MuLx10** [comment]
@rcurtin the function `GenerateNoisySines` calls itself leading to infinite recursion.

File: doc/tutorials/ann/ann.txt [hunk 1]
```diff
@@ -253,7 +253,9 @@ void GenerateNoisySines(arma::mat& data,
         arma::as_scalar(arma::randu(1) - 0.5) * noise;
     labels(1, sequences + seq) = 1;
   }
-
+}
```
Comment (rcurtin) [inline doc/tutorials/ann/ann.txt]: Can you put a blank line between the end of `GenerateNoisySines()` and `RNNModel()`?

Comment (MuLx10) [reply doc/tutorials/ann/ann.txt]: Sure

**Comment by ShikharJ** [comment]
@MuLx10 Thanks for the contribution.

**Pull request merged**